packages/plugin-core/src/commands/CreateDailyJournal.ts

```diff
@@ -21,22 +49,197 @@ export class CreateDailyJournalCommand extends CreateNoteWithTraitCommand {
     this.key = DENDRON_COMMANDS.CREATE_DAILY_JOURNAL_NOTE.key;
   }
 
-  override execute(opts: CommandOpts): Promise<void> {
+  override async execute(opts: CommandOpts): Promise<ExecuteData> {
     const config = this._extension.getDWorkspace().config;
     const journalConfig = ConfigUtils.getJournal(config);
+    const maybeDailyVault = journalConfig.dailyVault;
+    const vault = maybeDailyVault
+      ? VaultUtils.getVaultByName({
+          vaults: this._extension.getEngine().vaults,
+          vname: maybeDailyVault,
+        })
+      : undefined;
+    let isFirstTime = false;
+    let isSchemaCreated = false;
+    let isTemplateCreated = false;
 
-    if (_.isUndefined(journalConfig.dailyVault)) {
-      return super.execute({ ...opts });
-    } else {
-      const dailyVault = journalConfig.dailyVault;
-      const vault = dailyVault
-        ? VaultUtils.getVaultByName({
-            vaults: this._extension.getEngine().vaults,
-            vname: dailyVault,
-          })
-        : undefined;
-
-      return super.execute({ ...opts, vaultOverride: vault });
+    const metaData = MetadataService.instance().getMeta();
+    // Only create default schema/template if running Daily Journal for first time
```

Comment:
We're going to run into an issue here with existing users - we don't know if they've created Daily Journals in the past.  I'm thinking to make the experience less jarring for existing users, that we don't do this auto template creation for anyone who is an existing user, irrespective of whether they've run create daily journal in the past (as we just don't know). 

I think we can do this by adding an additional check here, something like `if MetadataService.firstInstallTime < May 31st, 2022` THEN don't auto-create the template.  

Ideally, we have a prop in metadata svc like `DendronVersionOnFirstInstall` so that we can do a version based check instead of a time based check.

Reply:
Ah good point. I can do the time based check while we come up with something better later